packages/authentication/spfx/tsconfig.base.json

```
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "importHelpers": true,
+        "noEmitOnError": true,
+        "moduleResolution": "node",
+        "strict": true,
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "forceConsistentCasingInFileNames": true,
+        "declaration": true,
+        "declarationMap": true,
+        "composite": true,
+        "paths": {
+            "@microsoft/kiota-abstractions": [
+                "../../abstractions"
+            ],
+        },
+
+        "skipLibCheck": true, // Otherwise, raises different issues with SPFx dependencies types (i.e: cannot find IInternalAadTokenProvider)
```

```suggestion
        "skipLibCheck": true
```
should be fine from the documentation, thanks for pointing out